docs/services/data/_index.md

```diff
@@ -43,56 +43,58 @@ Or learn more about Viam's data management features, which can be broken down in
 
 ## On-device data management
 
-Manage data directly on your robot, with configurable data capture from various components, automated recovery after interruptions, and secure cloud synchronization for efficient data storage and management.
+The data management service:
+
+- Automatically captures data at the rate you specify.
+  If your machine restarts, data capture resumes automatically.
+- Securely transfers data to the cloud at the frequency you define.
```

Comment:
I'd use "sync" or "upload' rather than 'transfer' for sake of consistency w other places we talk about it in docs

Reply:
Good call
```suggestion
- Securely syncs data to the cloud at the frequency you define.
```